README.md

```diff
@@ -15,46 +11,121 @@ hvPlot <img src="/doc/_static/logo.png" width="50" height="50">
+```
+
+Import the hvplot extension for your data source and optionally set the plotting backend
+
+```python
+import hvplot.pandas
+# Optional: import hvplot
+# Optional: hvplot.extension('matplotlib') or hvplot.extension('plotly')
+```
+
+Use the `.hvplot` API as you would use the Pandas `.plot` API.
+
+```python
+df.hvplot()
+```
+
+[<img src="https://github.com/MarcSkovMadsen/awesome-panel-assets/blob/master/images/hvPlot/hvplot-intro-plot.gif?raw=true" style="max-height:300px;border-radius:2%;">](https://hvplot.holoviz.org/user_guide/index.html)
+
+To get started check out the [Installation Instructions](#installation-instructions) and the [User Guide.](https://hvplot.holoviz.org/user_guide/index.html).
+
+## How to make your data pipeline  `.interactive`
+
+Just add `.interactive()` and replace your normal arguments with [Panel](https://panel.holoviz.org) widgets.
+
+```python
+import panel as pn
+pn.extension()
 
+df.interactive(width=600).head(n=pn.widgets.IntSlider(start=1, end=5, value=3))
 ```
+
+[<img src="https://github.com/MarcSkovMadsen/awesome-panel-assets/blob/master/images/hvPlot/hvplot-intro-interactive.gif?raw=true" style="max-height:300px;border-radius:2%;">](https://hvplot.holoviz.org/user_guide/Interactive.html)
+
+
+To get started check out the [`.interactive` Guide](https://hvplot.holoviz.org/user_guide/Interactive.html).
+
+## How to find documentation from your notebook or editor
+
+To see the available arguments for a specific `kind` of plot run
+
+```python
+import hvplot
+hvplot.help(kind='scatter')
+```
+
+In a notebook or ipython environment the usual
+
+- `help` and `?` will provide you with documentation.
+- `TAB` and `SHIFT+TAB` completion will help you navigate.
+
+## Support & Feedback
+
+| Type                     | Channel                                              |
+| ------------------------ | ------------------------------------------------------ |
+| 🚨&nbsp; **Bug Reports**       | <a href="https://github.com/holoviz/hvplot/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+label%3Abug+sort%3Areactions-%2B1-desc+" title="Open Bug Report"><img src="https://img.shields.io/github/issues/holoviz/hvplot/bug.svg?label=bug"></a>                                 |
+| 🎁&nbsp; **Feature Requests**  | <a href="https://github.com/holoviz/hvplot/issues?q=is%3Aopen+is%3Aissue+label%3Afeature+sort%3Areactions-%2B1-desc" title="Open Feature Request"><img src="https://img.shields.io/github/issues/holoviz/hvplot/feature.svg?label=feature%20request"></a>                                 |
+| 👩‍💻&nbsp; **Usage Questions**   |  <a href="https://discourse.holoviz.org/" title="Open Support Request"> <img src="https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.holoviz.org"></a> |
+| 📢&nbsp; **Announcements**  | <a href="https://twitter.com/HoloViz_org" title="Follow hvPlot on Twitter"><img src="https://img.shields.io/twitter/follow/HoloViz_org.svg?style=social&label=Follow"> |
```

Comment:
That section seems redundant compared with the first table.

Reply:
Maybe. 

- I'm not sure its clear to users that they can click the buttons on the first table. Its also a mix of icons that are only "kpis" and icons for navigation.
- A lot of users don't know what to go to github with and what to go to discourse with. This tables makes it clearer.

:-)

Reply:
And then I like the idea of having a link at the top of the document called Support.

![image](https://user-images.githubusercontent.com/42288570/182687610-800d3845-6f55-4dd1-9f9a-a4dccc6dc23a.png)

What is in this "menu" should be the most expected things users are looking for.

Reply:
Its not that important. Should I remove it @maximlt ? If yes should we include the twitter link in the top table (I think so 😄 )?

Reply:
Yes please I'd appreciate if you could remove the `Support and Feeback` section, which I think doesn't bring much to the table as the support link is already in the first table, and hvPlot doesn't (yet) have its own tweeter account.

Reply:
I just added Gitter and a note about the HoloViz call as feedback from Sophia Yang.

Reply:
I like the support table. It's what users care about: where to get help.  I think the support table is more valuable than the first table. Users don't care about test coverage, releases, and so on...

Reply:
Yes adding the Twitter link is a great idea!

Reply:
Now we have been exploring and discussing a bit. Could you update me on what I should do to resolve this one @maximlt ? Thanks.

Reply:
We're currently focusing on making the next bug fix release of hvPlot and this PR isn't going to be part of it, since as whenever it is merged it is going to make the README directly better :)

I still need to have a look at the recent changes you've made!